index.html

```diff
@@ -3555,6 +3555,21 @@ <h3>
 Signatures on <a>DID documents</a> are optional. <a>DID method</a>
 specifications SHOULD explain and specify their implementation if applicable.
         </p>
+
+        <div class="note" role="note" id="issue-container-generatedID-8">
+          <div role="heading" class="note-title marker" id="h-note-7" 
+          aria-level="4"><span>Note</span></div><p class="">
+Through the use of digital signatures a controller can assert proof of control 
+which can be used in a variety of different contexts. By asserting proof of 
+control of the identifier, the controller is asserting proof of ownership of 
+the public key which may not be the same as the subject of the DID. However, 
```

Comment:
Public key is never subject of a DID. Is subject of a DID actually related in any way to the proof of control?

Reply:
Yes, the controller (who needs to prove control - usually with a digital signature) has the authority to assert who the DID identifies as the subject. @jandrieu comments in #269 goes into better detail about the relationship between the two.

Reply:
This particular statement is confusing: "public key may not be the same as the subject of the DID." It does not really help to clarify the relationship between subject and controller. Public key and a thing identified by a DID are completely different entities. Perhaps you wanted to say that subject of a did may or may not have a control over did?

Reply:
Ahhh I see what you're pointing out now. I'll give an attempt to reword this a bit based on the way you put this.